server/src/main/java/io/envoyproxy/controlplane/server/DiscoveryRequestStreamObserver.java

```diff
@@ -22,6 +22,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * {@code DiscoveryRequestStreamObserver} is provides the base implementation for XDS stream handling.
```

Comment:
DiscoveryRequestStreamObserver provides. Without `is`

Reply:
good catch